### PR TITLE
Replace MouseButton1Click with Activated

### PR DESCRIPTION
**Pull request by Flikter**
## Changes

Replaces MouseButton1Click with Activated. Also removes unneeded WaitForChild calls.

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by Flikter** [comment]
Any update on this?

File: content/en-us/ui/text-filtering.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ local function getTextObject(message, fromPlayerId)
 	end)
 	if success then
 		return textObject
-	elseif errorMessage then
-		print("Error generating TextFilterResult:", errorMessage)
+	else
```
Comment (ahyatt-rbx) [inline content/en-us/ui/text-filtering.md:108]: this `else` isnt necessary as the function returns within the if statement

**Review by ahyatt-rbx — DISMISSED** [review]
Thanks for updating this, `Activated` should be used in favor of `MouseButton1Click`

File: content/en-us/ui/text-filtering.md (hunk 1 above)
Comment (Flikter) [reply content/en-us/ui/text-filtering.md]: I've removed it.

**Review by aetter — APPROVED** [review]

**Pull request merged**